package.json

```diff
@@ -46,13 +46,15 @@
     "start:detach": "npm run start -- --detach",
     "stop": "docker compose down",
     "type-check:all": "npm run type-check --workspaces --if-present",
-    "update:all": "npm update --save --if-present"
+    "update:all": "npm update --save --if-present",
+    "validate:yml": "npx yaml-lint **/*.yml"
```

Comment:
`lint:yml`?

Reply:
This script is validating the YML syntax only and not linting it.